ts/features/itwallet/presentation/details/components/ItwPresentationCredentialCard.tsx

```diff
@@ -48,9 +58,12 @@ const ItwPresentationCredentialCard = ({ credential }: Props) => {
   };
 
   const { backgroundColor } = getThemeColorByCredentialType(
-    credential.credentialType
+    credential.credentialType,
+    isL3Credential
```

Comment:
Why not you used `supportsL3Design` here? We can reduce repetitions by using the same hook for badge rendering and colors. What do you think?

```suggestion
    isL3Credential
```